charts/incubator/kubernetes-reflector/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+env:
+  ES_Serilog__MinimumLevel__Default: Information
+  ES_Reflector__Watcher__Timeout: ""
+  ES_Reflector__Kubernetes__SkipTlsVerify: "false"
+
+operator:
+  register: true
+
+portal:
+  open:
+    enabled: false
+
+rbac:
+  main:
+    enabled: true
+    primary: true
+    clusterWide: true
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - "configmaps"
+      - "secrets"
+      verbs:
+      - "get"
+      - "list"
+      - "watch"
+      - "create"
+      - "update"
+      - "patch"
+      - "delete"
+    - apiGroups:
+      - ""
+      resources:
+      - "namespaces"
+      verbs:
+      - "watch"
+      - "list"
+
+serviceAccount:
+  main:
+    enabled: true
+    primary: true
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+
+podOptions:
+  automountServiceAccountToken: true
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: http
+              path: /healthz
+            readiness:
+              type: http
+              path: /healthz
+            startup:
+              type: http
+              path: /healthz
+
+service:
+  # Only used for health probes
+  main:
+    ports:
+      main:
+        port: 25080
```

Comment:
You can omit it in that case, just set the `port:` property in each probe manually.
No need to have a service for probes!